### PR TITLE
WSTEAM1-1159: Uploader Text Input Enforced Colours Border Fix

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Resolves JIRA [WSTEAM1-1159](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1159)

Overall changes
======
Adds a border for enforced colours mode on the text inputs of the uploader form

Code changes
======

- Adds border colour using canvasText system colour for enforced colours
- Removes browser applied defaul outline for enforced colours
- Updates snapshots

Testing
======
1. Open Firefox or Edge on windows
2. Visit http://localhost:7081/somali/send/u130092370?renderer_env=test
3. Enables custom colours in firefox, or enable high contrast mode in windows
4. See text inputs using text colour as the border colour

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Review by amoore108 — APPROVED** [review]

**Review by Isabella-Mitchell — APPROVED** [review]
Checked on Firefox. Can also check on edge/windows if needed.

**Comment by greenc05** [comment]
Looking good just need to check on Windows High Contrast mode if someone hasn't then no need for test.

File: ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export default {
       padding: `${spacings.FULL}rem`,
       ...fontVariants.sansRegular,
       ...fontSizes.pica,
+      [mq.HIGH_CONTRAST]: { borderColor: 'canvasText', outline: '0' },
```
Comment (greenc05) [inline ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts]: Are you able to change the name of this to 'mq.FORCED_COLORS' in this PR @HarveyPeachey?

Comment (HarveyPeachey) [reply ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts]: I'd rather have it in a seperate PR for seperation of concern due to it being changed in multiple places. I can fire up a quick PR with the change in it

Comment (greenc05) [reply ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts]: Thanks Harvey 😄

Comment (HarveyPeachey) [reply ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts]: Here ya go: https://github.com/bbc/simorgh/pull/11791

**Pull request merged**